examples/mongocxx/index.cpp

```diff
@@ -37,9 +37,7 @@ int main(int, char**) {
     }
 
     // Create a single field index.
-    {
-        db["restaurants"].create_index(make_document(kvp("cuisine", 1)), {});
-    }
+    { db["restaurants"].create_index(make_document(kvp("cuisine", 1)), {}); }
```

Comment:
Is this from clang-format?

Reply:
Yes, this patch was failing lint for some reason?